src/lesson7/task3/Main.java

```diff
@@ -0,0 +1,13 @@
+package lesson7.task3;
+
+public class Main {
+    public static void main(String[] args) {
+        Point first_line_first_dot = new Point(1, 1);
```

Comment:
в джаві використовують камел кейс в більшості випадків 
```suggestion
        Point firstLineFirstDot = new Point(1, 1);
```